package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "globalize-webpack-plugin",
-  "version": "2.1.1",
+  "version": "3.0.0",
```

Comment:
You might want to update the [README.md](https://github.com/rxaviers/globalize-webpack-plugin/blob/master/README.md) to document changes for Webpack4.

Reply:
Yeap Mike. I went ahead and updated documentation accordingly.